### DISCUSSION TITLE
Practical series 1 submissions

### DISCUSSION THREAD

**Discussion by richardreeve** [IBAHCM/RPiR#533] [Announcements]
Hi @SBOHVM/rpir22 - can I remind everyone that we'd really like submissions to be complete before the class today so we can start practical series 2 straight away. Note that there are three people whose submissions are only drafts, which normally only happens by accident, so can everyone log into moodle and check that they have submitted asap. Thanks.

**Comment by AnnaKinghorn** [comment]
Hi Richard, there is no submission link for me on moodle - I’m assuming this is because I’m only auditing?
